packages/components/src/ui/item-group/use-item.ts

```diff
@@ -34,20 +30,14 @@ export function useItem( props: PolymorphicComponentProps< Props, 'div' > ) {
 
 	const size = sizeProp || contextSize;
 
-	const as = asProp || action ? 'button' : 'div';
-
-	const classes = cx(
-		action && styles.unstyledButton,
-		styles.itemSizes[ size ] || styles.itemSizes.medium,
-		styles.item,
-		spacedAround && styles.spacedAround,
-		className
-	);
+	const as = ( asProp || action ? 'button' : 'div' ) as ElementType;
 
 	return {
 		as,
-		className: classes,
 		role,
+		size,
+		spacedAround,
+		action,
```

Comment:
`action` is a valid HTML attribute, and therefore when it is passed down to `ItemView` React assigned an invalid value of `"true"|"false"` to the `action` attribute.

We should rename this property so that it doesn't conflict with any existing HTML attributes, maybe `hasAction` ?

Reply:
Yeah, `isAction` makes sense to me and follows our boolean naming agreement 🙂